### DISCUSSION TITLE
How can i change date/time format in AuditInfo

### DISCUSSION THREAD

**Discussion by HoenesDK** [oqtane/oqtane.framework#3302] [Q&A]
I can't find anywhere to change the Date/Time format in AuditInfo.

It is always shown as 'MMM dd yyyy HH:mm:ss'
![AuditInfo](https://github.com/oqtane/oqtane.framework/assets/51191152/86bb4f79-1a9f-41f0-a2a6-1a34f59f2819)

**Comment by leigh-pointer** [comment]
@HoenesDK you need to set the DateTimeFormat parameter

```
[Parameter]
public string DateTimeFormat { get; set; } = "MMM dd yyyy HH:mm:ss";
```

**Reply by HoenesDK** [reply]
Yes, but how can i change it for the standard admin modules ?

**Reply by leigh-pointer** [reply]
there is no global setting for Date Time format.

@sbwalker should something like this be implemented.  I'm happy to do it?

**Reply by sbwalker** [reply]
@leigh-pointer wouldn't it be better to infer this from the User's culture settings?

**Reply by leigh-pointer** [reply]
@sbwalker that is probably the neatest way to do it. We would need to cover all date properties and columns.

**Reply by sbwalker** [reply]
yes, its an invasive change which will affect a lot of UI components so I would prefer to wait until after 4.0.4 (which I plan to release within a few days)

**Reply by leigh-pointer** [reply]
@sbwalker did a little work on this get get a head start.  The CurrentCulture does change the date "G" format but what is required is an actual format i.e "dd-MMM-yyyy hh:mm" or "MMM, dd yy h:mm" etc.
